root/types.js

```diff
@@ -1,20 +1,40 @@
-// @flow
-// Copyright (C) 2017 MetaBrainz Foundation
-//
-// This file is part of MusicBrainz, the open internet music database,
-// and is licensed under the GPL version 2, or (at your option) any
-// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+/*
+ * @flow
+ * Copyright (C) 2017 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
 
 
-// Types are in alphabetical order.
-//
-// The definitions in this file are intended to model the output of the
-// TO_JSON methods under lib/MusicBrainz/Server/Entity/; those are precisely
-// how data is serialized for us.
+/*
+ * Types are in alphabetical order.
+ *
+ * The definitions in this file are intended to model the output of the
+ * TO_JSON methods under lib/MusicBrainz/Server/Entity/; those are precisely
+ * how data is serialized for us.
+ */
+
+declare type AreaT = CoreEntityT & {
+    typeName: string;
+    containment?: AreaT[];
```

Comment:
These should be indented two spaces. (We don't support Flow in our eslint setup yet.)